pkg/config/standard_names.go

```diff
@@ -0,0 +1,28 @@
+package config
+
+// StandardJMXIntegrations is the list of standard jmx integrations
+var StandardJMXIntegrations = map[string]struct{}{
+	"activemq":    {},
+	"activemq_58": {},
+	"cassandra":   {},
+	"jmx":         {},
```

Comment:
I think this prefix should be `jvm` instead of `jmx`. I don't think `jmx` should be listed here (do we have any jmxfetch integration that prefixes metrics with `jmx`?)

Reply:
You're right, nice catch.